Classes/ViewHelpers/CalcViewHelper.php

```diff
@@ -43,27 +43,37 @@
 class CalcViewHelper extends AbstractViewHelper
 {
 
+    /**
+     * initialize arguments
+     */
+    public function initializeArguments()
+    {
+        $this->registerArgument('val1', 'string', 'first value', true);
+        $this->registerArgument('val2', 'string', 'second value', true);
+        $this->registerArgument('operator', 'string', 'operator', false, '+');
+    }
+
     /**
      * Return result of calculation
-     *
-     * @param string $val1 first value
-     * @param string $val2 second value
-     * @param string $operator operator
-     * @return string
+     * @return float|int
      */
-    public function render($val1, $val2, $operator = '+')
+    public function render()
```

Comment:
What do you think about using renderStatic() instead?

Reply:
Thats a good idea. Changed.